slycot/src/AB08NX.f

```diff
@@ -199,8 +199,8 @@ SUBROUTINE AB08NX( N, M, P, RO, SIGMA, SVLMAX, ABCD, LDABCD,
       INTEGER           ILAENV
       EXTERNAL          ILAENV
 C     .. External Subroutines ..
-      EXTERNAL          DLAPMT, DLARFG, DLASET, DLATZM, DORMQR, DORMRQ,
-     $                  MB03OY, MB03PY, XERBLA
+      EXTERNAL          DLAPMT, DLARFG, DLASET, SLCT_DLATZM, DORMQR,
```

Comment:
Also possible. I came across this in the LAPACK license:
----------------------------------------------------------------------
Like all software, it is copyrighted. It is not trademarked, but we do ask the following:

If you modify the source for these routines we ask that you change the name of the routine and comment the changes made to the original.

We will gladly answer any questions regarding the software. If a modification is done, however, it is the responsibility of the person who modified the routine to provide support.
------------------------------------------------------------------

This makes it clearer we are using our own copy. The six-letter limit for fortran variable names is only relevant for FORTRAN 4 or 5, I think it was already dropped for FORTRAN 77